src/api/app/models/bs_request.rb

```diff
@@ -57,7 +57,7 @@ class SaveError < APIException
 
   after_update :send_state_change
 
-  def save!
+  def save!(args = {})
```

Comment:
Calling << on a CollectionProxy calls our overwritten save, which then calls super... One of the methods expects an argument the other doesn't => ArgumentError